packages/terra-table/tests/jest/TableRowContent.test.jsx

```diff
@@ -7,9 +7,3 @@ it('should render a default table row content', () => {
   const tableRowContent = shallow(defaultTableRowContent);
   expect(tableRowContent).toMatchSnapshot();
 });
-
-it('should render a table row content with maximum height', () => {
```

Comment:
Why did we remove this test?

Reply:
Same as above

It was actually your recommendation. It is a stale test left by [this pull request removing max height tests](https://github.com/cerner/terra-core/pull/238)